cluster/definition.go

```diff
@@ -139,6 +140,12 @@ func (d Definition) NodeIdx(pID peer.ID) (NodeIdx, error) {
 
 // VerifySignatures returns true if all config signatures are fully populated and valid. A verified definition is ready for use in DKG.
 func (d Definition) VerifySignatures() error {
+	// Skip signature verification for definition versions earlier than v1.3. This is because v1.3 introduced a breaking change to how EIP712
+	// signatures are calculated. https://github.com/ObolNetwork/charon/issues/1203
+	if semver.Compare(d.Version, v1_3) == -1 {
+		return nil
+	}
```

Comment:
best not introduce additional libraries without consulting with the team, please remove and rather use simple comparison: `if d.Version==v1_0 || d.Version==v1_1 || d.Version==v1_2`.

Reply:
Also, as mentioned earlier please error if signatures are present in this case, i.e., require empty signatures

Reply:
can extract the above condition into a function `supportEIP712Sigs`